test/dt_arith.c

```diff
@@ -1630,393 +1746,688 @@ test_derived_integer(void)
 }
 
 /*-------------------------------------------------------------------------
- * Function:    test_conv_int_1
- *
- * Purpose:    Test conversion of integer values from SRC to DST.
- *             These types should be any combination of:
+ * Function:    test_derived_complex
  *
- *             H5T_NATIVE_SCHAR    H5T_NATIVE_UCHAR
- *             H5T_NATIVE_SHORT    H5T_NATIVE_USHORT
- *             H5T_NATIVE_INT      H5T_NATIVE_UINT
- *             H5T_NATIVE_LONG     H5T_NATIVE_ULONG
- *             H5T_NATIVE_LLONG    H5T_NATIVE_ULLONG
+ * Purpose:     Tests user-defined and query functions of complex number
+ *              types.
  *
- * Return:    Success:    0
- *            Failure:    number of errors
+ * Return:      Success:    0
+ *              Failure:    number of errors
  *
  *-------------------------------------------------------------------------
  */
 static int
-test_conv_int_1(const char *name, hid_t src, hid_t dst)
+test_derived_complex(void)
```

Comment:
The git diff isn't great here; `test_derived_complex` was added and `test_conv_int_1` is just now below this function.